ibis/pyspark/client.py

```diff
@@ -0,0 +1,39 @@
+import ibis.expr.types as types
+from ibis.pyspark.compiler import translate
+from ibis.pyspark.operations import PysparkTable
+from ibis.spark.client import SparkClient
+
+from pyspark.sql.column import Column
+
+
+class PysparkClient(SparkClient):
```

Comment:
Can we change the naming convention to be `PySpark` instead of `Pyspark` as per the style here: https://spark.apache.org/docs/latest/api/python/index.html#?